pyctuator/pyctuator.py

```diff
@@ -30,6 +31,8 @@ def __init__(
             app_url: str,
             pyctuator_endpoint_url: str,
             registration_url: Optional[str],
+            registration_username: Optional[str] = None,
```

Comment:
shouldn't the `Auth` class be a part of pyctuator API, and pass in an instance of `Auth` instead of username and password? Doesn't this tie you to basic authentication in the future?

Reply:
I didn't propagate the `RegistrationEndpoint` to the `Pyctuator` constructor because this is part of a separate discussion Yanay started long ago regarding pyctuator configuration.
Lets discuss better interface today.